.github/workflows/common-js.yml

```
@@ -53,6 +55,7 @@ jobs:
     if: |
       github.event_name != 'pull_request'
         || needs.common-js-detect-changes.outputs.path-filter == 'true'
+        || github.event_name != 'schedule'
```

The change will make the job execute every time, no matter what the `event_name` was.
We wan't to exclude execution of the job for all workflows triggered by `schedule` and all workflows triggered by `pull_request` which have `path-filter` == 'false'.

```suggestion
      ( github.event_name != 'pull_request'
        && github.event_name != 'schedule`)
        || needs.common-js-detect-changes.outputs.path-filter == 'true'
```

---

Hm, but I think this is still not right, cause in case `path-filter` is `true` then the workflow will be executed for `schedule` or `pull_request` event, which we don't want. Question: do we want to execute this flow for all events (except `pull_request` & `schedule`) only when `path-filter` is `true`?

---

The `path-filter` can be true only for `pull_request` triggering event (see line [27](https://github.com/keep-network/local-setup/pull/119/files/a20ee756a0eb500873f1fa1a6e67aa8857f9611c#diff-bcd9044f56238812dac6dfb72691a009b42a5b0d37ca6f1307533baa25749082R27)). We won't have it equal `tru` if workflow is dispatched by different event than `pull_request`.

Generally, we want to execute job for:
 - all workflow dispatches
 - all pushes on `main` if change affects specified files (the filtering of paths is handled by lines 9-10)
 - all pull_requests if changes affect specified files (i.e. `path-filter == 'true'`)

---

> The path-filter can be true only for pull_request triggering event

I missed that ^ part, makes sense now. Changed here: 1968d81